docs/pages/docs/reference/telemetry.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Privacy Policy (Telemetry)"
+description: "Keystone Collects anonymous telemetry events when running `keystone dev` - Learn what is sent, what the information is used for, and how to opt-out"
+---
+
+## Summary
```

Comment:
```suggestion

```
We can start directly with content without a title unless you think this title is important